packages/core/src/http-handler.ts

```diff
@@ -60,6 +62,22 @@ export class HttpHandler {
     return requestContext;
   }
 
+  protected _bindGetFromContext(context: Context): void {
+    context.bind('getFromContext').toDynamicValue(() => {
+      return async (key: string) => {
+        return await context.get(key);
+      };
+    });
+  }
+
+  protected _bindBindElement(context: Context): void {
+    context.bind('bindElement').toDynamicValue(() => {
+      return async (key: string, value: ValueOrPromise<BoundValue>) => {
```

Comment:
I am using the term `Element` as per @ritch 's definition of objects created inside a running `Sequence`